### ISSUE TITLE
avro-ipc 1.11 not compatible with Jakarta 9/10

### ISSUE THREAD

**Issue by ppalaga** [apache/camel-quarkus#4462]
`org.apache.avro.ipc.ResponderServlet` from avro-ipc 1.11 extends `javax.servlet.http.HttpServlet`
This blocks the migration of `camel-quarkus-avro-rpc` to Camel 4/Quarkus 3.

**Comment by ppalaga** [comment]
Blocked by https://issues.apache.org/jira/browse/AVRO-3728

**Comment by jamesnetherton** [comment]
There's also this which would help to avoid Servlets:

https://issues.apache.org/jira/browse/AVRO-3166

**Comment by ppalaga** [comment]
> There's also this which would help to avoid Servlets:
> 
> https://issues.apache.org/jira/browse/AVRO-3166

Yes, exactly, I mentioned it in [AVRO-3728](https://issues.apache.org/jira/browse/AVRO-3728)
